client/my-sites/domains/domain-management/email/index.jsx

```diff
@@ -159,21 +160,14 @@ class Email extends React.Component {
 	}
 
 	addEmailForwardingCard() {
+		const { domains, selectedDomainName, selectedSite, translate } = this.props;
+		const domain = getEligibleDomain( selectedDomainName, domains );
```

Comment:
Why are we retrieving the first domain name that is eligible to G Suite here? Shouldn't we retrieve the first domain name that is eligible to email forwarding instead? According to the [support page](https://en.support.wordpress.com/email-forwarding/), email forwarding is also subject to the plan attached to the site. I wonder if we allow email forwarding for mapped domains.